External/Plugins/AS3Context/Context.cs

```diff
@@ -979,6 +979,19 @@ protected override void InitTopLevelElements()
             topLevel.Members.Sort();
         }
 
+        public override string GetDefaultValue(string type)
+        {
+            if (string.IsNullOrEmpty(type) || type == features.voidKey) return null;
+            switch (type)
+            {
+                case "int":
+                case "uint": return "0";
+                case "Number": return "NaN";
+                case "Boolean": return "false";
+                default: return "null";
```

Comment:
For features.dynamicKey "undefined" is ok
But I doubt that it's good for an empty type